client/src/pages/Bookings.js

```diff
@@ -0,0 +1,223 @@
+import Button from "react-bootstrap/Button";
+import Col from "react-bootstrap/Col";
+import Form from "react-bootstrap/Form";
+import Row from "react-bootstrap/Row";
+import DatePicker from "react-datepicker";
+import "react-datepicker/dist/react-datepicker.css";
+import "bootstrap/dist/css/bootstrap.min.css";
+import axios from "axios";
+import React, { useState } from "react";
+import { Template } from "../components/Template";
+
+export const Bookings = () => {
```

Comment:
Well strucktured and clean code. Naming conventions are clear and easy to understand. Functionality are well thought of.